Dockerfile

```diff
@@ -97,3 +103,13 @@ EXPOSE 3000
 
 ENTRYPOINT [ "bin/docker-entrypoint" ]
 CMD [ "rails", "server" ]
+
+# ------------------------------------------------------------------------------
+# test
+# ------------------------------------------------------------------------------
+
+FROM web AS test
```

Comment:
As mentioned in the PR description, can you add something to the Readme so we don't forget? Also, might be a good idea to add a script to `bin` that builds the docker container with the relevant flags

Reply:
I imagine we'll cover this as part of the CD story, I'll mention it in the readme though